### PR TITLE
Edit profile features and typos

### PR DESCRIPTION
**Pull request by DevChima**

### CODE REVIEW

File: src/ussd_registration.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ go.app = (function() {
   var moment = require("moment");
   var Choice = vumigo.states.Choice;
   var ChoiceState = vumigo.states.ChoiceState;
+  //var PaginatedState = vumigo.states.PaginatedState;
```
Comment (rudigiesler) [inline src/ussd_registration.js]: Could we just remove this line, instead of commenting it out?

File: test/ussd_registration.test.js [hunk 2]
```diff
@@ -526,13 +550,176 @@ describe("state_profile", function() {
         ].join("\n")
       })
       .run();
-    });
+    }); */
```
Comment (rudigiesler) [inline test/ussd_registration.test.js]: Should these blocks be commented out?

**Review by rudigiesler — COMMENTED** [review]
Just two comments about commenting out instead of deleting. Otherwise it looks fine to me. I'm not familiar with what the user flow should be, so I'm not able to check if it follows the flow that it's supposed to, but the code looks fine.

File: src/ussd_registration.js (hunk 1 above)
Comment (DevChima) [reply src/ussd_registration.js]: It shouldn't be there. I'll remove it

**Pull request merged**